### PR TITLE
Fix/swtch 1089 submit onchain for asset

### PR DESCRIPTION
**Pull request by jrhender**
Adds test for on-chain role for asset

### CODE REVIEW

File: test/claimsTests/enrollmentClaimsTests.ts [hunk 1]
```diff
@@ -100,7 +105,17 @@ export function enrollmentClaimsTests() {
...
+    await enrolAndIssue({ subjectDID: userDID, requesterDID: userDID });
+  });
+
+  test("asset enrollment by issuer of type DID", async () => {
+    const assetAddress = await userIam.registerAsset();
+    const assetDID = `did:${Methods.Erc1056}:${assetAddress}`;
+    await enrolAndIssue({ subjectDID: assetDID, requesterDID: userDID });
+  });
+
+  async function enrolAndIssue({ subjectDID, requesterDID }: { subjectDID: string, requesterDID: string }) {
```
Comment (JGiter) [inline test/claimsTests/enrollmentClaimsTests.ts:117]: I suggest to remove `requesterDID` since only user request enrollment

Comment (JGiter) [inline test/claimsTests/enrollmentClaimsTests.ts:117]:
```suggestion
  async function enrolAndIssue({ subjectDID }: { subjectDID: string }) {
```

File: test/claimsTests/enrollmentClaimsTests.ts [hunk 2]
```diff
@@ -111,7 +126,7 @@ export function enrollmentClaimsTests() {
 
     expect(message).toHaveProperty("id");
     expect(message).toHaveProperty("token");
-    expect(message).toMatchObject({ requester: subjectDID, registrationTypes });
+    expect(message).toMatchObject({ requester: requesterDID, registrationTypes });
```
Comment (JGiter) [inline test/claimsTests/enrollmentClaimsTests.ts:129]:
```suggestion
    expect(message).toMatchObject({ requester: userDID, registrationTypes });
```

**Comment by JGiter** [comment]
The reason of this bug is old version of ew-reg. I've added commit with updates

File: test/claimsTests/enrollmentClaimsTests.ts (hunk 1 above)
Comment (jrhender) [reply test/claimsTests/enrollmentClaimsTests.ts:117]: Thanks, great point

**Review by JGiter — APPROVED** [review]

**Pull request merged**